contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -26,9 +26,21 @@ contract AlignedLayerServiceManager is ServiceManagerBase, BLSSignatureChecker {
     uint256 internal constant _THRESHOLD_DENOMINATOR = 100;
 
     // STRUCTS
+    enum DASolution {
+        Calldata,
+        EigenDA,
+        Celestia
+    }
+
+    struct TaskDA {
+        DASolution solution;
+        bytes proof; // Proof bytes for calldata -  BatchHeaderHash for EigenDA - Commitment for Celestia
```

Comment:
small nit, I think it would be better if this field could have a more generic name like `DAPayload` or something similar. What do you think?

Reply:
Agree, changed on 89788f258aae54a058ef12ed83bb4cb75bbb8b55